pmd-xml/src/main/resources/category/xml/errorprone.xml

```diff
@@ -30,7 +30,7 @@ An XML CDATA section begins with a &lt;!CDATA[ marker, which has only one [, and
         </properties>
         <example>
 <![CDATA[
-An extra [ looks like &lt;!CDATA[[]]&gt;, and an extra ] looks like &lt;!CDATA[]]]&gt;.
+An extra [ looks like &lt;![CDATA[[]]&gt;, and an extra ] looks like &lt;![CDATA[]]]&gt;.
```

Comment:
See https://github.com/pmd/pmd/issues/1666#issuecomment-502348118 for suggestions for the examples.

Reply:
We could maybe also improve the description a bit...

Reply:
I'll finish this

Reply:
Sorry for losing momentum, was busy with other stuff. I've read your comments in the issue regarding cdata so they were not worthless. :)